tests/unit/platform_apis_test.c

```diff
@@ -24,6 +26,22 @@ CTEST_DATA(platform_api)
 
 CTEST_SETUP(platform_api)
 {
+   // This test exercises error cases, so even when everything succeeds
+   // it generates lots of "error" messages.
+   // By default, that would go to stderr, which would pollute test output.
+   // Here we ensure those expected error messages are only printed
+   // when the caller sets the VERBOSE env var to opt-in.
+   if (Ctest_verbose) {
+      platform_set_log_streams(stdout, stderr);
+      CTEST_LOG_INFO("\nVerbose mode on.  This test exercises an error case, "
+                     "so on sucess it "
+                     "will print a message that appears to be an error.\n");
+   } else {
+      FILE *dev_null = fopen("/dev/null", "w");
+      ASSERT_NOT_NULL(dev_null);
+      platform_set_log_streams(dev_null, dev_null);
+   }
```

Comment:
This block will be simplified with just call to one function when PR #514 checks-in to `/main`.